src/LanguageServer/Test/CompletionTests.cs

```diff
@@ -123,6 +123,63 @@ class oar(list):
                 .And.HaveInsertTextFormat(InsertTextFormat.PlainText);
         }
 
+        [DataRow(PythonLanguageVersion.V37)]
+        [DataRow(PythonLanguageVersion.V36)]
+        [DataTestMethod]
+        public async Task OverrideInit3X(PythonLanguageVersion version) {
+            const string code = @"
+class Test():
+    def __
+";
+            var analysis = await GetAnalysisAsync(code, version);
+            var cs = new CompletionSource(new PlainTextDocumentationSource(), ServerSettings.completion);
+            var result = cs.GetCompletions(analysis, new SourceLocation(3, 10));
+
+            result.Should().HaveItem("__init__")
+                .Which.Should().HaveInsertText($"__init__(self, *args, **kwargs):{Environment.NewLine}    super().__init__(*args, **kwargs)")
+                .And.HaveInsertTextFormat(InsertTextFormat.PlainText);
+        }
+
+        [DataRow(PythonLanguageVersion.V26, "value")]
```

Comment:
Seems to use same "value" in both data cases

Reply:
The function signature remained the same between the two. The parameter name actually seems to come from the list class. Between 36 and 37, the function parameter changed to object.